### DISCUSSION TITLE
Image platform does not match host platform

### DISCUSSION THREAD

**Discussion by avadiax** [akvorado/akvorado#1218] [Q&A] [answered]
Hi,

I need help with the following issue. I have installed Akvorado on a Raspberry Pi 5 using Docker v2. As I start spinning up using "docker compose up -d", I noticed that there is an issue with spinning up akvorado-geoip container as it complaint that the image platform does not match the Raspberry Pi 5. Will appreciate any guidance what are the steps/commands I should take to spin up the image that matches the platform. Thanks.

>[+] Running 13/13
> ✔ Network akvorado_default                                                                                                                             Created              0.1s
> ✔ Container akvorado-geoip-1                                                                                                                           Started              1.1s
> ✔ Container akvorado-zookeeper-1                                                                                                                       Started              1.1s
> ✔ Container akvorado-akvorado-orchestrator-1                                                                                                           Started              1.1s
> ✔ Container akvorado-akvorado-inlet-1                                                                                                                  Started              1.2s
> ✔ Container akvorado-redis-1                                                                                                                           Started              1.1s
> ✔ Container akvorado-traefik-1                                                                                                                         Started              0.9s
> ✔ Container akvorado-clickhouse-1                                                                                                                      Started              1.3s
> ✔ Container akvorado-akvorado-conntrack-fixer-1                                                                                                        Started              0.5s
> ✔ Container akvorado-kafka-1                                                                                                                           Started              1.9s
> ! geoip The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested                      0.0s
> ✔ Container akvorado-akvorado-console-1                                                                                                                Started              1.9s
> ✔ Container akvorado-kafka-ui-1                                                                                                                        Started              2.5s

**Comment by vincentbernat** [comment, marked as answer]
This image seems only to be built for amd64. The easiest workaround would be to build it locally on your Raspberry Pi. Checkout this repository: https://github.com/ipinfo/docker. Then, `docker build . -t ipinfo/ipinfo-db:latest`.

**Comment by vincentbernat** [comment]
This should be fixed in future versions with 7c4f68af934d.

**Reply by avadiax** [reply]
OK thanks. I will refrain from building myself and wait for the next version so as not to manage my own local version.

**Reply by vincentbernat** [reply]
It's really a one-time operation. You won't need to manage it anymore.
